app/controllers/members_controller.rb

```diff
@@ -0,0 +1,52 @@
+class MembersController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Ok, will change that. I will also fix the `GroupsController` to allow anyone to see a group details when I work on the group view page.